apps/marketplace/components/BriefChoice/BriefChoice.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import { Radio } from '@gov.au/control-input/lib/js/react.js'
```

Comment:
I wonder if this is easier in latest version

Reply:
I don't think so - it seems the latest uikit still points its package.json `main` attribute to the es5 code.